src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -24,9 +24,22 @@
     <ToolHostCmd Condition="'$(COMPLUS_InstallRoot)' != ''">(set COMPLUS_InstallRoot=) &amp; $(ToolHostCmd)</ToolHostCmd>
   </PropertyGroup>
 
+  <!-- If BUILDTOOLS_TOOLS_RUNTIME is set, then we will use that as the tool and test runtime, else we will restore it from NuGet. -->
+  <Target Name="UseBuildToolsRuntimeFromEnvVariable"
```

Comment:
I don't think any of this is necessary. If folks want to use a different runtime they can simply set ToolRuntimePath to point at the directory they have setup. They would also need to copy the ToolRuntime.semaphore file there as well.